database/factories/InsegnamentoFactory.php

```diff
@@ -10,8 +10,8 @@
 $factory->define(Insegnamento::class, function (Faker $faker) {
 
     $maxCorsoDiStudiProgressive = CorsoDiStudi::max('id');
-    $idModulo = array($faker->numberBetween(2000, 10000), null)[rand() % 2]; 
-    $nomeModulo = $idModulo != null ? 'laboratorio' : null;  
+    $idModulo = array($faker->numberBetween(2000, 10000), null)[rand() % 2];
+    $nomeModulo = $idModulo != null ? 'laboratorio' : null;
```

Comment:
Spazi?

Reply:
me lo ha fatto un plugin in automatico, mi pare si chiami `whitespaces`, rimuove gli spazi inutili